mobile/src/main/java/org/openhab/habdroid/model/ParsedState.java

```diff
@@ -212,6 +218,28 @@ private static NumberState parseAsNumber(String state, String format) {
         return null;
     }
 
+    private static Location parseAsLocation(String state) {
+        String[] splitState = state.split(",");
+        // Valid states are either "latitude,longitude" or "latitude,longitude,elevation",
+        // (we ignore elevation in the latter case)
+        if (splitState.length == 2 || splitState.length == 3) {
+            try {
+                Location l = new Location("openhab");
+                l.setLatitude(Float.valueOf(splitState[0]));
+                l.setLongitude(Float.valueOf(splitState[1]));
+                l.setTime(System.currentTimeMillis());
```

Comment:
The time when the location was changed in openHAB isn't the time when the user starts the app. I suggest to not set the time here.

Reply:
I basically only set the time to make the debug output look nicer, as `Location.toString()` prints some question marks if it isn't set ;-)